docs/sources/setup-grafana/configure-grafana/feature-toggles/index.md

```diff
@@ -197,6 +197,7 @@ Experimental features might be changed or removed without prior notice.
 | `dataplaneAggregator`                       | Enable grafana dataplane aggregator                                                                                                                                                                                                                                               |
 | `newFiltersUI`                              | Enables new combobox style UI for the Ad hoc filters variable in scenes architecture                                                                                                                                                                                              |
 | `lokiSendDashboardPanelNames`               | Send dashboard and panel names to Loki when querying                                                                                                                                                                                                                              |
+| `alertingPrometheusRulesPrimary`            | Uses Prometheus rules as primary source of truth for Grafana for Ruler-enabled datasources                                                                                                                                                                                        |
```

Comment:
```suggestion
| `alertingPrometheusRulesPrimary`            | Uses Prometheus rules as the primary source of truth for ruler-enabled data sources                                                                                                                                                                                        |
```

Reply:
I'm not sure whether we can take out Grafana here as it's self-evident as it sounded odd?